.github/workflows/publish-latest-v3.yml

```diff
@@ -0,0 +1,85 @@
+name: Publish to latest / v3
+
+on:
+  push:
+    branches: [ui-react/v3]
+
+permissions:
+  contents: write # Used to push tags to GitHub
+
+jobs:
+  setup:
+    runs-on: ubuntu-latest
+    outputs:
+      has-changesets: ${{ steps.has-changesets.outputs.has-changesets }}
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+        with:
+          # This makes Actions fetch all Git history so that Changesets can generate changelogs with the correct commits
+          fetch-depth: 0
+      - name: Check if repo has unconsumed changesets
+        id: has-changesets
+        uses: ./.github/actions/has-changesets
+
+  # build-test:
+  #   runs-on: ubuntu-latest
+  #   needs: setup
+  #   environment: ci
+  #   if: ${{ needs.setup.outputs.has-changesets != 'true' }}
+  #   steps:
+  #     - name: Checkout repo
+  #       uses: actions/checkout@v3
+  #     - name: Add Amplify CLI
+  #       run: yarn global add @aws-amplify/cli
+  #     - name: Get CLI versions
+  #       id: cli-version
+  #       run: echo "::set-output name=version::$(amplify --version)"
+  #     - name: Create or restore environments cache
+  #       id: environments-cache
+  #       uses: actions/cache@v2
+  #       with:
+  #         path: canary/environments/**/aws-exports.js
+  #         key: ${{ runner.os }}-canary-environments-${{ steps.cli-version.outputs.version }}-${{ hashFiles('canary/environments/**/amplify/**') }}
+  #     - name: Pull down AWS environments
+  #       if: steps.environments-cache.outputs.cache-hit != 'true'
+  #       run: yarn pull
+  #       env:
+  #         AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+  #         AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+  #       working-directory: ./canary
+  #     - name: Setup canary apps against @next
+  #       run: yarn setup:next
```

Comment:
couldn't we target the v2 tag instead?

Reply:
The problem is that build-test needs to happen after v3 tag release. We could either:
1. run build-test after publish, and do a quick follow up if anything goes wrong, or
2. have an `unstable-v3` tag that we can test with before the release, or
3. we run build tests manually

Reply:
Since this is a one-off, I'm ok with number 3

Reply:
Just to clarify. The step would be:

1) run this publish workflow for every commit on the `ui-react/v3`
2) once the publish workflow is done, manually run the build-test

If that's right, would it be hard to remember to run the test? (I, with poor memory, would prefer doing the test automatically 🥴  )

Reply:
Oh sorry, merged before I saw the comment.

1 is a bit different, we'll have a version packages PR for v3. 

1. "Version Packages" PR is prepared.
2. Engineer does build tests
3. Engineer merges the "Version Packages" PR